Dockerfile.base

```diff
@@ -2,7 +2,7 @@ FROM node:16-slim
 WORKDIR /app
 
 # Setup pnpm package manager
-RUN npm install -g pnpm@7.11.0
+RUN npm install -g pnpm
```

Comment:
You should pin the version to avoid accidentally installing pnpm 9 and breaking the project